docker/ci/test_webapp.sh

```diff
@@ -16,6 +16,9 @@ wait_for_runit "$container"
 check_service_status "$container" /etc/service/nginx
 check_service_status "$container" /home/simplified/service/uwsgi
 
+# Wait for UWSGI to be ready to accept connections.
+timeout 120s grep -q 'WSGI app .* ready in [0-9]* seconds' <(docker logs "$container" -f 2>&1)
+
```

Comment:
Could also be helpful to emit a message (in addition to the exit code) for the "health check" and "app server running" error cases below.